### PR TITLE
[BH-1690] Add information about the connection to the PC

### PR DESCRIPTION
**Pull request by lukasmas**
Added text with information about the connection to the main window when the USB cable is connected to the PC

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

**Review by dawidwojtas-mudita — APPROVED** [review]
Good job ;)

File: products/BellHybrid/apps/common/src/widgets/BellConnectionStatus.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+{
+    constexpr auto usb_connected_status = "app_bellmain_usb_status_connected";
+} // namespace
+
+namespace gui
+{
+    BellConnectionStatus::BellConnectionStatus(Item *parent) : HBox(parent)
+    {
+        statusText = new Text(parent);
+        statusText->setMaximumSize(350U, 102);
```
[the remark this answers is not in the document]
Comment (lukasmas) [reply products/BellHybrid/apps/common/src/widgets/BellConnectionStatus.cpp:19]: All three done :)

File: products/BellHybrid/apps/application-bell-main/ApplicationBellMain.cpp [hunk 2]
```diff
@@ -72,6 +74,12 @@ namespace app
             requestShutdownWindow(gui::window::name::bell_battery_shutdown);
             return actionHandled();
         });
+
+        connect(typeid(sdesktop::usb::USBConfigured), [&](sys::Message *msg) -> sys::MessagePointer {
+            LOG_ERROR("Configured USB");
```
Comment (Maciej-Mudita) [inline products/BellHybrid/apps/application-bell-main/ApplicationBellMain.cpp:79]: are you sure there should be LOG_ERROR here?

**Review by Maciej-Mudita — APPROVED** [review]

File: products/BellHybrid/apps/application-bell-main/ApplicationBellMain.cpp (hunk 2 above)
Comment (lukasmas) [reply products/BellHybrid/apps/application-bell-main/ApplicationBellMain.cpp:79]: There should be no log :)
Good catch!

Comment (lukasmas) [reply products/BellHybrid/apps/application-bell-main/ApplicationBellMain.cpp]: Removed

**Pull request merged**